docs_src/microservices/device/V3Migration.md

```diff
@@ -271,15 +271,32 @@ TBD
 
 This section is specific to changes made only to **Device USB Camera**
 
-See [Top Level V3 Migration Guide](../../../V3TopLevelMigration) for details applicable to all EdgeX services and [All Device Services](#all-device-services) section above for details applicable to all EdgeX device services. . 
-
-TBD
+See [Top Level V3 Migration Guide](../../../V3TopLevelMigration) for details applicable to all EdgeX services and [All Device Services](#all-device-services) section above for details applicable to all EdgeX device services.
 
 #### Configuration
 
-TBD
+All USB camera rtsp streams need authentication by default. To properly configure credentials for the stream, follow the below instructions.
 
-#### Device Profile
+1. Set the credentials in the `configuration.yaml` 
 
-TBD
+    !!! example - "Snippet from configuration.yaml"
+        ```yaml
+        Writable:
+            LogLevel: "INFO"
+            InsecureSecrets:
+              rtspauth:
+                SecretName: rtspauth
+                SecretData:
+                  username: "<user>"
+                  password: "<pass>"
+        ```
+    
+    To use curl commands to set these values, see [here](./supported/device-usb-camera/walkthrough/deployment.md#add-credentials-for-the-rtsp-stream)
 
+    
+1. You can configure the authentication server to run from a different port by editing the externalAuthenticationURL value in the Dockerfile. To disable authentication entirely, comment out the externalAuthenticationURL line in the Dockerfile.
```

Comment:
Same as above `Dockerfile` pointing to the actual location will be more useful for the user.